{"name": "Проверка: нельзя создать двух одинаковых курьеров", "status": "passed", "description": "Повторный POST запрос на /api/v1/courier с такими же данными вернет 409 Conflict", "start": 1742218746999, "stop": 1742218747549, "uuid": "89838886-7193-4752-a425-3579e6ce41e5", "historyId": "676cdb776c8034ff3c263e12946fdffd", "testCaseId": "676cdb776c8034ff3c263e12946fdffd", "fullName": "tests.test_create_courier.TestCreateCourier#test_courier_creation_with_the_same_data_return_409_conflict", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "DESKTOP-FO1K8TG"}, {"name": "thread", "value": "14656-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}